content/6.artiel-release.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Artiel's release
+description: Skyra Project's Fun bot just released!
+date: 2023-06-06
+tags: [first-release]
+author: Aura
+image: artiel
+---
+
+**Artiel is Skyra Project's Fun bot**, he comes with a growing database of meme templates powered by our custom-made
+[Meme Template Generator](https://memes.skyra.pw).
+
+::card{type="info"}
+
+Right now, the meme template database will start small, but will soon be filled with lots of templates. We tried to use
+an external database, but not only they are locked down and require payment for accessing the list, but they also come
+with a lot of duplicates, something we aim to fix with our tool.
```

Comment:
```suggestion
Right now, the meme template database will start small, but we aim to soon provide lots of templates.
```

The whole second sentence is unnecessary. No one will care about such fine details.